### ISSUE TITLE
[FEAT] Support loongarch

### ISSUE THREAD

**Issue by CarterLi** [fastfetch-cli/fastfetch#1204]
```
system type		: generic-loongson-machine

processor		: 0
package			: 0
core			: 0
global_id		: 0
CPU Family		: Loongson-64bit
Model Name		: Loongson-3A6000-HV
CPU Revision		: 0x00
FPU Revision		: 0x00
CPU MHz			: 2500.00
BogoMIPS		: 5000.00
TLB Entries		: 2112
Address Sizes		: 48 bits physical, 48 bits virtual
ISA			: loongarch32 loongarch64
Features		: cpucfg lam ual fpu lsx lasx crc32 lvz lbt_x86 lbt_arm lbt_mips
Hardware Watchpoint	: yes, iwatch count: 8, dwatch count: 4

processor		: 1
package			: 0
core			: 0
global_id		: 1
CPU Family		: Loongson-64bit
Model Name		: Loongson-3A6000-HV
CPU Revision		: 0x00
FPU Revision		: 0x00
CPU MHz			: 2500.00
BogoMIPS		: 5000.00
TLB Entries		: 2112
Address Sizes		: 48 bits physical, 48 bits virtual
ISA			: loongarch32 loongarch64
Features		: cpucfg lam ual fpu lsx lasx crc32 lvz lbt_x86 lbt_arm lbt_mips
Hardware Watchpoint	: yes, iwatch count: 8, dwatch count: 4

processor		: 2
package			: 0
core			: 1
global_id		: 2
CPU Family		: Loongson-64bit
Model Name		: Loongson-3A6000-HV
CPU Revision		: 0x00
FPU Revision		: 0x00
CPU MHz			: 2500.00
BogoMIPS		: 5000.00
TLB Entries		: 2112
Address Sizes		: 48 bits physical, 48 bits virtual
ISA			: loongarch32 loongarch64
Features		: cpucfg lam ual fpu lsx lasx crc32 lvz lbt_x86 lbt_arm lbt_mips
Hardware Watchpoint	: yes, iwatch count: 8, dwatch count: 4

processor		: 3
package			: 0
core			: 1
global_id		: 3
CPU Family		: Loongson-64bit
Model Name		: Loongson-3A6000-HV
CPU Revision		: 0x00
FPU Revision		: 0x00
CPU MHz			: 2500.00
BogoMIPS		: 5000.00
TLB Entries		: 2112
Address Sizes		: 48 bits physical, 48 bits virtual
ISA			: loongarch32 loongarch64
Features		: cpucfg lam ual fpu lsx lasx crc32 lvz lbt_x86 lbt_arm lbt_mips
Hardware Watchpoint	: yes, iwatch count: 8, dwatch count: 4

processor		: 4
package			: 0
core			: 2
global_id		: 4
CPU Family		: Loongson-64bit
Model Name		: Loongson-3A6000-HV
CPU Revision		: 0x00
FPU Revision		: 0x00
CPU MHz			: 2500.00
BogoMIPS		: 5000.00
TLB Entries		: 2112
Address Sizes		: 48 bits physical, 48 bits virtual
ISA			: loongarch32 loongarch64
Features		: cpucfg lam ual fpu lsx lasx crc32 lvz lbt_x86 lbt_arm lbt_mips
Hardware Watchpoint	: yes, iwatch count: 8, dwatch count: 4

processor		: 5
package			: 0
core			: 2
global_id		: 5
CPU Family		: Loongson-64bit
Model Name		: Loongson-3A6000-HV
CPU Revision		: 0x00
FPU Revision		: 0x00
CPU MHz			: 2500.00
BogoMIPS		: 5000.00
TLB Entries		: 2112
Address Sizes		: 48 bits physical, 48 bits virtual
ISA			: loongarch32 loongarch64
Features		: cpucfg lam ual fpu lsx lasx crc32 lvz lbt_x86 lbt_arm lbt_mips
Hardware Watchpoint	: yes, iwatch count: 8, dwatch count: 4

processor		: 6
package			: 0
core			: 3
global_id		: 6
CPU Family		: Loongson-64bit
Model Name		: Loongson-3A6000-HV
CPU Revision		: 0x00
FPU Revision		: 0x00
CPU MHz			: 2500.00
BogoMIPS		: 5000.00
TLB Entries		: 2112
Address Sizes		: 48 bits physical, 48 bits virtual
ISA			: loongarch32 loongarch64
Features		: cpucfg lam ual fpu lsx lasx crc32 lvz lbt_x86 lbt_arm lbt_mips
Hardware Watchpoint	: yes, iwatch count: 8, dwatch count: 4

processor		: 7
package			: 0
core			: 3
global_id		: 7
CPU Family		: Loongson-64bit
Model Name		: Loongson-3A6000-HV
CPU Revision		: 0x00
FPU Revision		: 0x00
CPU MHz			: 2500.00
BogoMIPS		: 5000.00
TLB Entries		: 2112
Address Sizes		: 48 bits physical, 48 bits virtual
ISA			: loongarch32 loongarch64
Features		: cpucfg lam ual fpu lsx lasx crc32 lvz lbt_x86 lbt_arm lbt_mips
Hardware Watchpoint	: yes, iwatch count: 8, dwatch count: 4
```

Ref: https://github.com/loongarchlinux/extra/pull/11 @GaugeAndGravity

**Comment by CarterLi** [comment]
@GaugeAndGravity Can you test the latest dev build?

**Comment by GaugeAndGravity** [comment]
> @GaugeAndGravity Can you test the latest dev build?

Yes, it works fine for me!
```
❯ fastfetch --config all
                                         wyj@loong-arch
                                         --------------
                  -`                     OS: Arch Linux loongarch64
                 .o+`                    Host: Loongson-3A6000-HV-7A2000-NUC (20.5)
                `ooo/                    Bios (UEFI): Loongson-UDK2018-V4.0.05692-stable202402
               `+oooo:                   Bootmgr: ARCH - grubloongarch64.efi
              `+oooooo:                  Board: Loongson-3A6000-HV-7A2000-NUC (QA612N0 V1.1)
              -+oooooo+:                 Chassis: Desktop (Mini PC V1.1)
            `/:-:++oooo+:                Kernel: Linux 6.8.6-2
           `/++++/+++++++:               Init System: systemd 255.3-1-arch
          `/++++++++++++++:              Uptime: 1 day, 22 mins
         `/+++ooooooooooooo/`            Loadavg: 0.02, 1.03, 0.83
        ./ooosssso++osssssso+`           Processes: 182
       .oossssso-````/ossssss+`          Packages: 458 (pacman)
      -osssssso.      :ssssssso.         Shell: zsh 5.9
     :osssssss/        osssso+++.        Editor: vim 9.1
    /ossssssss/        +ssssooo/-        Monitor (KA240Y): 1920x1080 px - 517x323 mm (24.00 inches, 91.79 ppi)
  `/ossssso+/:-        -:/+osssso+-      LM: sshd 9.6p1 (TTY)
 `+sso+:-`                 `.-/+oso:     Cursor: Adwaita
`++:.                           `-/+/    Terminal: /dev/pts/1
.`                                 `/    Terminal Size: 144 columns x 37 rows (3744px x 1961px)
                                         Terminal Theme: #EBC4CB (FG) - #0C0D15 (BG) [Dark]
                                         CPU: Loongson-3A6000-HV loongarch64 (8) @ 2.50 GHz - 40.0°C
                                         CPU Cache (L1): 8x64.00 KiB (I), 8x64.00 KiB (D)
                                         CPU Cache (L2): 8x256.00 KiB (U)
                                         CPU Cache (L3): 16.00 MiB (U)
                                         CPU Usage: 0%
                                         GPU: Loongson Technology LLC Device 7A36 (VGA compatible)
                                         Memory: 2.74 GiB / 15.77 GiB (17%)
                                         Swap: Disabled
                                         Disk (/): 5.55 GiB / 237.97 GiB (2%) - btrfs
                                         Public IP: 131.91.7.56 (Boca Raton, US)
                                         Local IP (enp0s3f0): 192.168.247.202/24 fe80::faa0:3dff:fe81:3eb3/64 (f8:a0:3d:81:3e:b3)
                                         Wifi: down
                                         Date & Time: 2024-08-21 21:37:24
                                         Locale: en_US.UTF-8
                                         Users: wyj@192.168.247.201 - login time 2024-08-21 14:19:47
                                         Sound: Dummy Output (100%)
                                         Weather: +81°F - Partly cloudy (Boca Raton, Florida, United States)
                                         Network IO (enp0s3f0): 3.17 KiB/s (IN) - 6.88 KiB/s (OUT) *
                                         Disk IO (ZHITAI PC005 Active 256GB): 884.00 KiB/s (R) - 0 B/s (W)
                                         Physical Disk (ZHITAI PC005 Active 256GB): 238.47 GiB [SSD, Fixed] - 40.0°C
                                         Version: fastfetch 2.21.3-23 (loongarch)

```

**Comment by CarterLi** [comment]
Good
